charts/newrelic-prometheus-agent/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: newrelic-prometheus-agent
 description: A Helm chart to deploy Prometheus with New Relic Prometheus Configurator.
 type: application
-version: 0.1.1
+version: 0.2.0
```

Comment:
perhaps would be nice to merge this #142 before released so the link is added to the chart release note

Reply:
Approved!